.eslintrc

```diff
@@ -3,7 +3,7 @@
   "extends": "plugin:@theforeman/foreman/core",
   "rules": {
     "spellcheck/spell-checker": [
-      "warn",
+      "error",
```

Comment:
for me the missed spellchecks were always correct. I even merged one PR with spellcheck warnings, because it was false-possitive, so I think we should keep it on warn and add false-possitives to vocabulary in batches?

Reply:
I saw a few times that the spelling was wrong and it got accidentally merged because as a warning it's not much visible :)

Reply:
I would not want to force spell check on us, it just have so many false-possitives :shrug:

Reply:
I get your point, but I don't like passing warning and errors to the next PR, and I think it's really easy to add that false positive word to the eslintrc